### ISSUE TITLE
Fix the focus ring in modals

### ISSUE THREAD

**Issue by chasenlehara** [CCALI/A2JViewer#189]
In Chrome, [`:focus-visible`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible) is used to display an outline around buttons in the modal header:
<img width="616" alt="Visible focus ring in Chrome" src="https://user-images.githubusercontent.com/10070176/147600337-93868ae4-b701-4cb3-89ad-0b3146b4e97e.png">

In Safari (Version 15.1), the focus ring is not as visible: 
<img width="622" alt="Almost invisible focus ring in Safari" src="https://user-images.githubusercontent.com/10070176/147600401-83c4b254-81ff-493a-9921-0f19df6021a8.png">

**Comment by janebitovi** [comment]
I am at a loss. I have no idea why safari is showing this as blue instead of white. Everything in the dev tools is correct, it's just ignoring the CSS....
![safari doesn't work](https://user-images.githubusercontent.com/92131113/151072430-7ec5e8f7-3879-4611-8f20-6b1f0fca20d3.gif)

**Comment by janebitovi** [comment]
only a11y solution is to remove the outline in safari and use something like box-shadow to fake it instead
![safaribad](https://user-images.githubusercontent.com/92131113/151199173-ab2ad8b3-b4e6-48a2-a68d-7671deed19d3.gif)
gets the job done at least
